crates/swc_ecma_minifier/src/program_data.rs

```diff
@@ -540,69 +533,65 @@ impl VarDataLike for VarUsageInfo {
 }
 
 impl ProgramData {
-    #[allow(clippy::single_range_in_vec_init)]
-    pub(crate) fn expand_infected(
-        &self,
-        module_info: &ModuleInfo,
-        ids: FxHashSet<Access>,
-        max_num: usize,
-    ) -> Option<FxHashSet<Access>> {
-        let init =
-            HashSet::with_capacity_and_hasher(max_num, BuildHasherDefault::<FxHasher>::default());
-        ids.into_iter()
-            .try_fold(init, |mut res, id| {
-                let mut ids = Vec::with_capacity(max_num);
-                ids.push(id);
-                let mut ranges = vec![0..1usize];
-                loop {
-                    let range = ranges.remove(0);
-                    for index in range {
-                        let iid = ids.get(index).unwrap();
-
-                        // Abort on imported variables, because we can't analyze them
-                        if module_info.blackbox_imports.contains(&iid.0) {
-                            return Err(());
-                        }
-                        if !res.insert(iid.clone()) {
-                            continue;
-                        }
-                        if res.len() >= max_num {
-                            return Err(());
-                        }
-                        if let Some(info) = self.vars.get(&iid.0) {
-                            let infects = &info.infects_to;
-                            if !infects.is_empty() {
-                                let old_len = ids.len();
-
-                                // This is not a call, so effects from call can be skipped
-                                let can_skip_non_call = matches!(iid.1, AccessKind::Reference)
-                                    || (info.declared_count == 1
-                                        && info.declared_as_fn_decl
-                                        && !info.reassigned());
-
-                                if can_skip_non_call {
-                                    ids.extend(
-                                        infects
-                                            .iter()
-                                            .filter(|(_, kind)| *kind != AccessKind::Call)
-                                            .cloned(),
-                                    );
-                                } else {
-                                    ids.extend_from_slice(infects.as_slice());
-                                }
-                                let new_len = ids.len();
-                                ranges.push(old_len..new_len);
-                            }
-                        }
-                    }
-                    if ranges.is_empty() {
-                        break;
-                    }
-                }
-                Ok(res)
-            })
-            .ok()
-    }
+    //     #[allow(clippy::single_range_in_vec_init)]
```

Comment:
@Austaras Is there a reason you commented this out instead of removing?

Reply:
No. It's just a large chunk of code so I thought maybe it would be useful one day. But we could also retrieve it from git history.

Reply:
If so, can you remove them?

Reply:
Done.